### ISSUE TITLE
reduce the labels

### ISSUE THREAD

**Issue by idhamhalim** [paulbota/car-detection-darkflow#1]
is it possible to reduce the labels in the coco.name file , since we are only detecting car

**Comment by paulbota** [comment]
Yes it is. This file was taken from the original repo which is used to detect a lot of labels
